geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/SpatialTiledRasterLayer.scala

```diff
@@ -151,7 +151,12 @@ class SpatialTiledRasterLayer(
       rdd.metadata
     )
 
-    val _neighborhood = getNeighborhood(operation, neighborhood, param1, param2, param3)
+    val _neighborhood =
+      if (operation == Constants.SLOPE || operation == Constants.ASPECT)
+        getNeighborhood(operation, neighborhood, 1.0, 0.0, 0.0)
```

Comment:
Are we now going to overwrite the size/area of a `Neighborhood` when the user wants to use a `SLOPE` or `ASPECT` operation? If so, then we'll have to update the docs to reflect that.

Reply:
Hmm, actually `Slope` and `Aspect` are only defined for Square(1) and their implementations in GeoTrellis assume it, which makes them different from other focal operations.

That seems to imply that they should be treated different, perhaps with `.slope` and `.aspect` methods ?

Reply:
I see. In that case we should probably break them out into their own methods. That doesn't need to happen for this PR, but I think we should break them out before merging the `feature/vector-pipe` branch into `master`.